docs/distributions/nixos/installation.md

```diff
@@ -1,44 +1,215 @@
+- some experience with the command line
+- sufficient mental energy to find and fix errors
 
-    **Note**: You might want to leave a little part of your disk as a FAT32 partition to be able to transfer files easily between MacOS and Linux.
+## Installation Steps
+### Preparation
+Follow the [pre-installation guide](../../guides/preinstall.md). You may not be able to complete some steps without doing so, like [Partitioning](#partitioning).
 
-2. To connect to internet, you should load the firmware with these commands first:
+### Partitioning
+After booting to the live environment, partition the disk (usually `/dev/nvme0n1`) using `cfdisk` or other tools of your preference.
+If you have followed the pre-installation guide, there should be another partition created just for Linux.
 
-    ```sh
-    sudo mkdir -p /lib/firmware/brcm
-    sudo cp /mnt/boot/firmware/* /lib/firmware/brcm
-    sudo modprobe -r brcmfmac && sudo modprobe brcmfmac
-    ```
+- Remove that and allocate new partitions accordingly - there are no hard rules. Just make sure to not delete the EFI partition. See the [NixOS installation manual](https://nixos.org/manual/nixos/stable/#sec-installation-manual-partitioning-UEFI) for examples.
+- Format the newly provisioned partitions with `mkfs.<filesystem>` commands and `mkswap` a swap partition (if any).
+- Mount the partitions under `/mnt`, and create directories as necessary. Then, enable the swap partition by using `swapon`. You should finally have something that looks similar to this:
 
-    Then run `systemctl start wpa_supplicant` and then connect to internet using `wpa_cli`.
-    When running the commands, don't forget to change `/mnt/boot` to the place you mounted your ESP at (for example `/mnt/boot/efi`).
+	```console
+	$ lsblk
+	NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
+	loop0         7:0    0 972.5M  0 loop /nix/.ro-store
+	sda           8:0    1  14.5G  0 disk
+	├─sda1        8:1    1  1021M  0 part
+	└─sda2        8:2    1     3M  0 part
+	nvme0n1     259:0    0 465.9G  0 disk
+	├─nvme0n1p1 259:1    0   300M  0 part  /mnt/boot  # EFI partition
+	├─nvme0n1p2 259:2    0  83.9G  0 part             # APFS/macOS partition
+	├─nvme0n1p3 259:3    0     1G  0 part  /mnt/macos_share  # optional
+	├─nvme0n1p4 259:4    0     8G  0 part  [SWAP]     # Swap Partition
+	└─nvme0n1p5 259:5    0 372.8G  0 part  /mnt       # Linux Partition
+	```
 
-3. Generate your configuration using `sudo nixos-generate-config --root /mnt`.
-4. Edit `/mnt/etc/nixos/configuration.nix`:
-    * Add `"${builtins.fetchGit { url = "https://github.com/NixOS/nixos-hardware.git"; }}/apple/t2"` to `imports`.
-    * Copy the WiFi/Bluetooth firmware to `/mnt/etc/nixos/firmware/brcm` and add this snippet to your `configuration.nix`:
+	This is just an example and does not have to be done this way.
 
-        ```nix
-        hardware.firmware = [
-          (pkgs.stdenvNoCC.mkDerivation {
-            name = "brcm-firmware";
+!!! tip
```

Comment:
Is !!! tip valid?

Reply:
`!!! tip` is valid per [mkdocs-material's admonitions reference](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:tip).